src/openrct2.c

```diff
@@ -225,7 +225,11 @@ bool openrct2_initialise()
 		audio_init();
 		audio_get_devices();
 	}
-	language_open(gConfigGeneral.language);
+	if (!language_open(gConfigGeneral.language))
+	{
+		log_error("Failed to open language, exiting.");
+		exit(1);
```

Comment:
elsewhere in this function we return false on a fatal error. Also this should be `log_fatal` for this one